### PR TITLE
📦 NEW: Test PHPCS

### PR DESCRIPTION
**Pull request by ahmadbilaldev**

### CODE REVIEW

File: prism-languages.php [hunk 1]
```diff
@@ -62,5 +63,6 @@ function mkaz_code_syntax_block_get_supported_languages() {
 	 *
 	 * @param string $languages Array of languages supported
 	 */
+
```
Comment (theprismane) [inline prism-languages.php:66]: :no_entry_sign: **Error**: Whitespace found at end of line (*Squiz.WhiteSpace.SuperfluousWhitespace.EndLine*).

File: prism-languages.php [hunk 2]
```diff
@@ -4,6 +4,7 @@
  * Retrieve support languages
  * @return array Hash of lang => language
  */
+// Test PHPCS
 function mkaz_code_syntax_block_get_supported_languages() {
```
Comment (theprismane) [inline prism-languages.php:9]: :no_entry_sign: **Error**: Whitespace found at end of line (*Squiz.WhiteSpace.SuperfluousWhitespace.EndLine*).

File: prism-languages.php [hunk 3]
```diff
@@ -4,6 +4,7 @@
...
```
Comment (theprismane) [inline prism-languages.php:8]: :no_entry_sign: **Error**: You must use &quot;/**&quot; style comments for a function comment (*Squiz.Commenting.FunctionComment.WrongStyle*).

**Review by theprismane — CHANGES_REQUESTED** [review]
**phpcs** scanning turned up:

:no_entry_sign: 7 errors 

***

**hashes-api**-scanning skipped

File: prism-languages.php [hunk 4]
```diff
@@ -4,6 +4,7 @@
  * Retrieve support languages
  * @return array Hash of lang => language
  */
+// Test PHPCS
```
Comment (theprismane) [inline prism-languages.php:7]: :no_entry_sign: **Error**: Inline comments must end in full-stops, exclamation marks, or question marks (*Squiz.Commenting.InlineComment.InvalidEndChar*).

File: prism-languages.php [hunk 5]
```diff
@@ -4,6 +4,7 @@
  * Retrieve support languages
  * @return array Hash of lang => language
  */
```
Comment (theprismane) [inline prism-languages.php:6]: :no_entry_sign: **Error**: Missing @package tag in file comment (*Squiz.Commenting.FileComment.MissingPackageTag*).

Comment (theprismane) [inline prism-languages.php:6]: :no_entry_sign: **Error**: There must be exactly one blank line after the file comment (*Squiz.Commenting.FileComment.SpacingAfterComment*).

File: prism-languages.php [hunk 6]
```diff
@@ -4,6 +4,7 @@
  * Retrieve support languages
  * @return array Hash of lang => language
```
Comment (theprismane) [inline prism-languages.php:5]: :no_entry_sign: **Error**: There must be exactly one blank line before the tags in a doc comment (*Generic.Commenting.DocComment.SpacingBeforeTags*).

**Comment by ahmadbilaldev** [comment]
Success

**Pull request closed without merging**